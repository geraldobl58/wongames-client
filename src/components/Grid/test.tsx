import { render } from 'utils/test-utils'

import { Grid } from '.'

describe('<Grid />', () => {
  it('should render correctly', () => {
    const { container } = render(<Grid />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
