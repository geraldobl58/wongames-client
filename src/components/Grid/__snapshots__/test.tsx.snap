// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Grid /> should render correctly 1`] = `
.c0 {
  display: grid;
  grid-template-columns: repeat(auto-fit,minmax(25rem,1fr));
  grid-gap: 3.2rem;
  margin: 3.2rem 0;
}

<div
  class="c0"
/>
`;
